backend/clubs/models.py

```diff
@@ -1789,6 +1789,7 @@ class Cart(models.Model):
     owner = models.OneToOneField(
         get_user_model(), related_name="cart", on_delete=models.CASCADE
     )
+    checkout_context = models.CharField(max_length=8297, blank=True, null=True)
```

Comment:
Played around with [Cybersource's code sample](https://github.com/CyberSource/cybersource-rest-samples-python/blob/master/samples/UnifiedCheckout/generate-unified-checkout) for generating a capture context. Returned capture context was always 8297 chars in length. Couldn't find this documented anywhere though.

Reply:
b64 encoding adds a fixed overhead over what is likely to be a standard format...would be my best guess

Can you please add a comment explaining why it's 8297?

Reply:
Yup, makes sense. Added.